src/routes/(console)/project-[project]/overview/(components)/deleteBatch.svelte

```diff
@@ -0,0 +1,59 @@
+<script lang="ts">
+    import { page } from '$app/state';
+    import { base } from '$app/paths';
+    import { sdk } from '$lib/stores/sdk';
+    import { Dependencies } from '$lib/constants';
+    import { goto, invalidate } from '$app/navigation';
+    import Confirm from '$lib/components/confirm.svelte';
+    import { addNotification } from '$lib/stores/notifications';
+    import { Submit, trackEvent, trackError } from '$lib/actions/analytics';
+
+    export let showDelete = false;
+    export let keyIds: string[] = [];
+    export let keyType: 'api' | 'dev' = 'api';
+
+    const projectId = page.params.project;
+
+    const isApiKey = keyType === 'api';
+    const label = isApiKey ? 'API' : 'dev';
+    const slug = isApiKey ? 'keys' : 'dev-keys';
+    const event = isApiKey ? Submit.KeyDelete : Submit.DevKeyDelete;
+    const dependency = isApiKey ? Dependencies.KEYS : Dependencies.DEV_KEYS;
```

Comment:
We can move all this logic inside the `handleDelete` function since we use it only there

Reply:
Done.